bin/reth/src/commands/stage/unwind.rs

```diff
@@ -105,9 +123,14 @@ impl Command {
         let (tip_tx, tip_rx) = watch::channel(B256::ZERO);
         let executor = block_executor!(provider_factory.chain_spec());
 
-        let pipeline = Pipeline::builder()
-            .with_tip_sender(tip_tx)
-            .add_stages(
+        let builder = if self.offline {
+            Pipeline::builder().add_stages(
+                OfflineStages::new(executor, config.stages, PruneModes::default())
+                    .builder()
+                    .disable(reth_stages::StageId::SenderRecovery),
```

Comment:
why do we disable it?

Reply:
I guess "offline" isn't the best phrasing, but the purpose of the flag is mostly for benchmarking execution. Keeping sender recovery means once we've unwound, we can just start at execution.

Reply:
yeah I think this makes sense because it's reasonable to expect that if you want to keep headers+bodies then you also want recovered senders